cloth-diapers/cloth-diapers-basics.md

```diff
@@ -0,0 +1,296 @@
+  + The advantage is that Yes, they dry faster. But this also defeats the
+  purpose of an AIO.
+
+https://petitecrown.com/collections/pocket-diapers/products/trima
+<img width="542" alt="image" src="https://user-images.githubusercontent.com/4185328/169338055-bc6d9347-cf20-40e6-9b95-64499a089a38.png">
+
+
+
+
+### Exceptions to the (my) Rules
+#### AI2s
+Although I really don't like AI2s, there is one kind that made it into our
+stash that I have not sold on and will not be selling: Close Pop-in AI2s.
+
+Let me be clear: these are possibly the _most_ annoying AI2s on the market.
+Putting them together is a total nightmare of tiny poppers that are not only
+different from every other diaper on the market but _different from one side
+of the diaper to the other_ :fire:
+I find them impressively infuriating.
+
+BUT.
+
+For _months_ (months 3 - 8 to be precise), they were the only diapers that
+I would trust to not have a major :poop: leak when we left the house.
+They are Hellish to put together, but their fit is worth gold for a few months :gem:
+
+https://closeparent.com/collections/pop-in/products/single-printed-nappy-bamboo-2021
+<img width="498" alt="image" src="https://user-images.githubusercontent.com/4185328/170536819-27882f9e-4a3b-4e55-bece-9bb49099f8f6.png">
+
+
+### Transition Diapers
+There is an annoying stage when they are slightly too big for the majority of
+newborn diapers but slightly too small for OS diapers (causing issues with
+leaks), so there are a few brands of OS diapers that run small which are
+perfect for this time.
+
+In our case they were:
++ LighthouseKids Company Signature AIO (we still use these today quite happily)
++ Totsbots Easyfit Star AIO (another diaper we still use today)
++ Blumchen AIO
++ Tickle Tots AIOs
++ Baby Bare pockets
++ Close Pop-in AI2s (we dropped these as soon as we could)
+
+We no longer use the last 3 on this list but _I have not sold them_ because they
+were ***critical*** for the transition phase and I will use them again with
+other kids (mine or family/friends').
+
+
+
+
+## Things to Know
++ The amount of liquid your diapers will need to hold will not only change
+as the baby grows up, but also with the seasons - babies need a lot more
+absorbency in Winter than in Summer (because in Summer they release water in
+other ways)
++ It is common for you to have to keep letting out rise snaps (increasing the
+'height' of the diaper) until you find yourself with diapers at their biggest
+by the time your baby is 8 or 9 months old - fear not! This doesn't mean they
+are about to stop fitting. It's around this age that babies start moving more,
+learning to pull to stand and eventually walking so their body shape changes
+and they lean out. You'll often find that you may need to snap the rise back
+up again in the near future as this happens.
++ Never iron your inserts or any diaper paraphenalia as this decreases absorption.
```

Comment:
This should be in a "Care" section in the `laundry.md` ...

Reply:
Agree. This was mostly a place I put miscellaneous points that didn't fit into other sections yet 👍